### PR TITLE
Version 1.3.0 Parsley

### PR DESCRIPTION
**Pull request by j-wags**
Moved from #25 to work with files on a fork instead of a branch.

> `openff-1.3.0.offxml` and `openff_unconstrained-1.3.0.offxml` were added. Details of the version 1.3.0 can be found in [Version 1.3.0 "Parsley" release package](https://github.com/openforcefield/openforcefield-forcebalance/releases/tag/v1.3.0)

### CODE REVIEW

**Comment by j-wags** [comment]
Hm, so I think I'm going to hold off on the section version bumps for `Bonds` and `ProperTorsions` here. We're supposed to increment both of them to `version="0.4"`, but Bonds 0.4 will only be supported by OFFTK 0.8.0, which isn't even out yet. 

The spec version bumps come with a change in the default values for `fractional_bondorder_method` and `fractional_bondorder_interpolation`, but I _think_ that, since the current OFFXML files specify those explicitly, it should be OK to keep them at section version `0.3`. 

There are a lot of moving pieces to this, so while I'm pretty sure that this is OK to merge, I'm going to sleep on it so I don't make a mistake.

**Comment by j-wags** [comment]
OK. I'm leaving the `Bonds` and `ProperTorsions` section versions unchanged for this. The 0.4 section version bump can happen for the Sage release. 

The `fractional_bondorder_method` and `interpolation` fields ARE being changed from their previous values, since the previous `Bonds` sections actively specified settings that are incompatible with bond WBO work (basically a cargo-culted mistake that we've been carrying around in our FF template). 

I don't believe there's anything in the 0.3 section specs that should forbid the new `fractional_bondorder_X` values, however older versions of the toolkit (before 0.7.0) won't be able to load these. 

Basically, I've thought about this for a long time, and come to the same conclusion as what @hyejang did initially in #25. I'll consider this as basically an agreeing cross-check on the decision. It's a real honor to work on such a quality-focused team :-)
